workloads/files/workload-mastervertical-script-cm.yml

```diff
@@ -80,7 +80,8 @@ data:
       result_dir=${benchmark_results_dir}
     fi
     start_time=$(date +%s)
-    VIPERCONFIG=/tmp/mastervertical.yaml openshift-tests run-test "[Feature:Performance][Serial][Slow] Load cluster concurrently with templates [Suite:openshift]" | tee "${result_dir}/clusterloader.txt"
+    git clone https://github.com/cloud-bulldozer/snafu; pip install -r snafu/requirements.txt;
```

Comment:
Can we bake the dependencies into the container image being used by the workloads (https://github.com/openshift-scale/images/tree/master/scale-ci-workload)? This way, we can avoid installing them during the runtime for each scale test run.

Reply:
nice suggestion, https://github.com/openshift-scale/images/pull/25 has been opened for this. Once that's merged I will update this pr and remove the extra step of installing requirements.